composer.json

```diff
@@ -7,7 +7,7 @@
     "prefer-stable": true,
     "type": "project",
     "require": {
-        "php": "~5.4",
+        "php": "~5.4 | ~7.0",
```

Comment:
Why not `>=5.4` ? Is there anything in PHP 6 dev (that was never officially released) that we are afraid of?